### ISSUE TITLE
Multiproject shortcuts

### ISSUE THREAD

**Issue by shanshin** [Kotlin/kotlinx-kover#466]
## Motivation

Now, in order to apply and configure the plugin in all subprojects, you need to write a special code that you need to get to, or find in the documentation. 
Also, this code cannot be placed in quickstart, because it is quite large.

```
val rootProject = project

allprojects {
    apply("org.jetbrains.kotlinx.kover")
    
    rootProject.dependencies.add("kover", project(path))
    
    kover {
        useJacoco()
    }
}
```

## Solution
Introduce special function for apply Kover Plugin in subprojects

for kts script 

```
// for all projects in build
koverApply {
    useJacoco.set(true)

    // also
    disabled.set(false)
    jacocoVersion.set("0.8.8")
    excludeJavaCode.set(false)
    excludeInstrumentation { ... }
}

// for current and specified projects
koverApply(":impl", ":extra") {
    ...
}

// for all projects in build except specified
koverApply(exceptions = setOf(":api", ":util")) {
    ...
}
```
for Groovy

```
kotlinx.kover.gradle.plugin.dsl.Shortcuts.koverApply {
    ...
}
```

**Comment by shanshin** [comment]
Implemented in `0.8.0-Beta`.
This is not a production ready release, the plugin DSL is experimental and can be changed after receiving feedback.

**Comment by shanshin** [comment]
>  Shouldn't this be discouraged in the spirit of project isolation, which newer gradle versions are gravitating towards?

Yes, using `merge { ... }` block is incompatible with the project isolation. This is explicitly stated in the documentation, first of all these shortcuts are intended for those who do not use the project isolation and have already had a similar `subprojects { ... }` code.

Another way to solve this problem is currently being [prototyped](https://github.com/Kotlin/kotlinx-kover/issues/608).
